Adyen/UI/Form/Items/FormImageItem/FormImageItem.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// Simple form item that represent a single UImage element.
+@_spi(AdyenInternal)
+public class FormImageItem: FormItem, Hidable {
+
+    public var isHidden: AdyenObservable<Bool> = AdyenObservable(false)
+
+    public var subitems: [FormItem] = []
+
+    /// The name of the Image.
+    public var name: String
+
+    public var identifier: String?
+
+    /// The size of the Image.
+    public var size: CGSize
+
+    /// The style of the Image.
+    public var style: ImageStyle?
+    
+    public init(name: String,
+                size: CGSize? = nil,
+                style: ImageStyle? = nil,
+                identifier: String? = nil) {
+        self.name = name
+        self.size = size ?? .init(width: 46, height: 46)
+        self.style = style
+        self.identifier = identifier
+    }
+
+    public func build(with builder: FormItemViewBuilder) -> AnyFormItemView {
+         return FormImageView(item: self)
+    }
+}
+
+internal class FormImageView: FormItemView<FormImageItem> {
+
+    internal func renderImage() {
+        let imageView = UIImageView()
+        imageView.layer.borderColor = item.style?.borderColor?.cgColor
+        imageView.layer.borderWidth = item.style?.borderWidth ?? 0.0
```

Comment:
`borderWidth` is non-optional.

```suggestion
        imageView.layer.borderWidth = item.style?.borderWidth
```

Reply:
SideNote:

Maybe we can create [helper like we have for ButtonStyle](https://github.com/Adyen/adyen-ios/blob/06afadbce62d7c8e9fcf0ba41f9b563b5f14df55/Adyen/Helpers/UIButtonHelpers.swift) ?

Reply:
Here The item.style is optional that's why it is asking for default value.

Reply:
Ok, this is minor..

Reply:
Added UImageView helper. 6558657c4e8927667faf6b2c5596b827b171d93d